content/blog/hacking-build-process.md

```diff
@@ -0,0 +1,344 @@
+          readOnly: true
+      resources: {}
+    volumes:
+      - name: truststore-volume
+        secret:
+          secretName: kafka-truststore      
+  persistence:
+    postgresql:
+      secretRef:
+        name: sonataflow-psql-postgresql
+        userKey: postgres-username
+        passwordKey: postgres-password
+      serviceRef:
+        name: sonataflow-psql-postgresql
+        port: 5432
+        databaseName: sonataflow
+        databaseSchema: lock-flow
+```
+
+Now we can deploy the manifests to the cluster. The files are numbered for a reason - the configmap must be applied first, otherwise, the SonataFlow operator will generate an empty configmap for the workflow:
+
+```bash
+oc create -f manifests/01-configmap_lock-flow-props.yaml -n sonataflow-infra
+oc create -f manifests/02-sonataflow_lock-flow.yaml -n sonataflow-infra
+```
+
+Let's watch for the workflow pod status to verify it's running:
+
+```bash
+oc get pod -n sonataflow-infra -l app=lock-flow
+```
+
+You should see output like this:
+
+```
+NAME                        READY   STATUS    RESTARTS   AGE
+lock-flow-5cbd5dbdb-2g7wz   1/1     Running   0          13m
+```
+
+## Testing the Workflow
+
+Now we can test the workflow by emitting cloud events to trigger it. Note that in this example, we're not using the Orchestrator to invoke workflows, since it can only invoke workflows via HTTP endpoints. However, we can monitor the workflow's progress from the workflow runs tab.
+
+### Setting Up the Kafka Producer with TLS
+
+To invoke the workflow, we'll use Kafka's producer script. Since we're using TLS, we need to have the `truststore.jks` file available.
+
+> In this example, we're using Kafka installed on an OpenShift cluster with the Strimzi operator.
+
+First, copy or mount the `truststore.jks` file to the Kafka cluster broker pod. Then create a properties file at `/tmp/client-ssl.properties`:
+
+```bash
+security.protocol=SSL
+ssl.truststore.location=/tmp/truststore.jks
+ssl.truststore.password=password
+ssl.enabled.protocols=TLSv1.2,TLSv1.1,TLSv1
+ssl.protocol=TLS
+```
+
+### Triggering the Workflow
+
+Now invoke the Kafka producer:
+
+```bash
+./bin/kafka-console-producer.sh \
+  --broker-list kafka-cluster-3-kafka-bootstrap:9093 \
+  --topic lock-event \
+  --producer.config /tmp/client-ssl.properties
+```
+
+This command opens an interactive session where you can paste Kafka messages.
+
+#### Step 1: Send the Lock Event
+
+To start a workflow, first paste the content of the [lock-event](https://github.com/masayag/poc-kafka-logic-operator/blob/main/kafka-messages/lock-event.json):
+
+```json
+{
+  "specversion": "1.0",
+  "id": "db16ff44-5b0b-4abc-88f3-5a71378be171",
+  "source": "http://dev.local",
+  "type": "lock-event",
+  "datacontenttype": "application/json",
+  "time": "2025-03-07T15:04:32.327635-05:00",
+  "lockid": "03471a81-310a-47f5-8db3-cceebc63961a",
+  "data": {
+    "name": "The Kraken",
+    "id": "03471a81-310a-47f5-8db3-cceebc63961a"
+  }
+}
```

Comment:
did you try copy/pasting this to the producer script? Are the new lines correctly escaped so it does not send 1 message per line instead of 1 message for the whole block?